crates/idxdb-store/src/account/js_bindings.rs

```diff
@@ -27,27 +35,29 @@ extern "C" {
     #[wasm_bindgen(js_name = getAccountStorage)]
     pub fn idxdb_get_account_storage(storage_root: String) -> js_sys::Promise;
 
-    #[wasm_bindgen(js_name = getAccountAssetVault)]
-    pub fn idxdb_get_account_asset_vault(vault_root: String) -> js_sys::Promise;
+    #[wasm_bindgen(js_name = getAccountStorageMaps)]
+    pub fn idxdb_get_account_storage_maps(roots: Vec<String>) -> js_sys::Promise;
+
+    #[wasm_bindgen(js_name = getVaultAssets)]
+    pub fn idxdb_get_vault_assets(vault_root: String) -> js_sys::Promise;
```

Comment:
There is some minor inconsistency in this functions names i think, we should add/remove `Account` from both, basically having:
```suggestion
    #[wasm_bindgen(js_name = getAccountStorageMaps)]
    pub fn idxdb_get_account_storage_maps(roots: Vec<String>) -> js_sys::Promise;

    #[wasm_bindgen(js_name = getAccountVaultAssets)]
    pub fn idxdb_get_account_vault_assets(vault_root: String) -> js_sys::Promise;
```
Or 
```suggestion
    #[wasm_bindgen(js_name = getStorageMaps)]
    pub fn idxdb_get_storage_maps(roots: Vec<String>) -> js_sys::Promise;

    #[wasm_bindgen(js_name = getVaultAssets)]
    pub fn idxdb_get_vault_assets(vault_root: String) -> js_sys::Promise;
```

I prefer the second.